website/docs/rbs-support.md

```diff
@@ -224,24 +224,25 @@ understanding the cases where there is no suitable RBS replacement.
+| [Intersection type]    | `Foo & Bar`                                           | [`T.all(Foo, Bar)`](intersection-types.md)                                                                           |
+| [Optional type]        | `Foo?`                                                | [`T.nilable(Foo)`](nilable-types.md)                                                                                 |
+| [Untyped type]         | `untyped`                                             | [`T.untyped`](untyped.md)                                                                                            |
+| [Boolean type]         | `bool`                                                | [`T::Boolean`](class-types.md#booleans)                                                                              |
+| [Nil type]             | `nil`                                                 | [`NilClass`](class-types.md#nil)                                                                                     |
+| [Top type]             | `top`                                                 | [`T.anything`](anything.md)                                                                                          |
+| [Bottom type]          | `bot`                                                 | [`T.noreturn`](noreturn.md)                                                                                          |
+| [Void type]            | `void`                                                | [`void`](sigs.md#returns--void-annotating-return-types)                                                              |
+| [Generic type]         | `Foo[Bar]`                                            | [`Foo[Bar]`](generics.md)                                                                                            |
+| [Generic method]       | `[U] (U foo) -> U`                                    | [`type_parameters(:U).params(foo: T.type_parameter(:U)).returns(T.type_parameter(:U))`](generics.md#generic-methods) |
+| [Tuple type]           | `[Foo, Bar]`                                          | [`[Foo, Bar]`](tuples.md)                                                                                            |
+| [Shape type]           | `{ a: Foo, b: Bar }`                                  | [`{ a: Foo, b: Bar }`](shapes.md)                                                                                    |
+| [Proc type]            | `^(Foo) -> Bar`                                       | [`T.proc.params(arg: Foo).returns(Bar)`](procs.md)                                                                   |
+| [Block type]           | `{ -> R } -> R`<br>`? { -> R } -> R` (optional block) | [`T.proc.params(arg: Foo).returns(Bar)`](procs.md#optional-blocks)                                                   |
```

Comment:
While you can add a space, it's generally written without https://github.com/ruby/rbs/blob/227062421a7505cb1c3fd57ef8a12903e7404c96/core/class.rbs#L85

```suggestion
| [Block type]           | `{ -> R } -> R`<br>`?{ -> R } -> R` (optional block) | [`T.proc.params(arg: Foo).returns(Bar)`](procs.md#optional-blocks)                                                   |
```